static/app/views/starfish/queries/useIndexedSpans.tsx

```diff
@@ -15,15 +15,23 @@ interface Filters {
   [key: string]: string;
 }
 
-export const useIndexedSpans = (
-  filters: Filters,
-  sorts: Sort[] = [{field: 'timestamp', kind: 'desc'}],
-  limit: number = DEFAULT_LIMIT,
-  enabled: boolean = true,
-  referrer: string = 'use-indexed-spans'
-) => {
+export const useIndexedSpans = ({
+  filters,
+  sorts = [{field: 'timestamp', kind: 'desc'}],
```

Comment:
IMO don't set defaults here! None of the other hooks do, and it's a bit of a surprising behaviour. Same for limit, and referrer! We _want_ the referrers to be set from the parent

Reply:
👍 dropped defaults from sort, limit, and referrer! Made some updates to the existing callers to support this